homeassistant/components/shelly/const.py

```diff
@@ -72,6 +72,18 @@
     "button3": 3,
 }
 
+RGB_EFFECTS = {
+    0: "Off",
+    1: "Meteor shower",
+    2: "Gradual change",
+    3: "Breath",
+    4: "Flash",
+    5: "On/off gradual",
+    6: "Red/green change",
```

Comment:
We better use this list only for Shelly Color Bulb (`SHBLB-1`) and make another one for all other devices
```suggestion
    1: "Meteor Shower",
    2: "Gradual Change",
    3: "Breath",
    4: "Flash",
    5: "On/Off Gradual",
    6: "Red/Green Change",
```

Reply:
Thx for noticing.

I copied the name from Shelly App/Cloud. 
Will update to capital letter in next commit